tools/dify_hugegraph/provider/hugegraph_query.py

```
@@ -0,0 +1,32 @@
+        try:
+            base_url = f"http://{host}:{port}"
+            session = requests.Session()
+
+            if username and password:
+                session.auth = (username, password)
+
+            # Test connectivity by getting graph info
+            response = session.get(f"{base_url}/graphs/{graph}")
+            response.raise_for_status()
+
+        except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)